### PR TITLE
10x faster of classes

### PR DESCRIPTION
**Pull request by stephenhky**

### CODE REVIEW

File: shorttext/classifiers/embed/nnlib/VarNNEmbedVecClassification.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import shorttext.utils.classification_exceptions as e
 from shorttext.utils import tokenize
 from shorttext.utils.compactmodel_io import CompactIOMachine
+from typing import Union, List
```
Comment (ragrawal) [inline shorttext/classifiers/embed/nnlib/VarNNEmbedVecClassification.py:12]: @stephenhky : add Dict over here

Comment (stephenhky) [reply shorttext/classifiers/embed/nnlib/VarNNEmbedVecClassification.py]: Added

Comment (ragrawal) [reply shorttext/classifiers/embed/nnlib/VarNNEmbedVecClassification.py]: you missed "Any".

Comment (stephenhky) [reply shorttext/classifiers/embed/nnlib/VarNNEmbedVecClassification.py]: Added

**Comment by ragrawal** [comment]
Nice everything passed

**Pull request merged**